TwinePlatform/twine-monolith
# ✨ Add FAQs page to Dashboard

fixes #254 

### Changes
- Create FAQs page
- Install `react-accessible-accordion`
- Change `Navbar` layout to widen the right hand side
- Add `FAQs` link to the `Navbar`

Notes:
Differs from design in one way: the chevron to indicate "drop-down" functionality is on the left of each header instead of on the right. Because of the way this is done in `react-accessible-accordion`, styling the latter would take more time than it's worth IMO.

### Testing Requirements
- [x] Dashboard app
  - Log in
  - Observe `FAQs` link in the Navbar
  - Click on `FAQs` link
  - Compare with [design](https://xd.adobe.com/spec/d9d42346-b996-4a50-587b-12a19033107c-afe5/screen/a85a626f-0fd0-4879-81c9-d47fc0b1bca7/FAQs-page)

### Release Requirements
- Release when ready

### Manual Deployment
